src/main/java/io/stargate/sgv2/jsonapi/service/operation/tables/DropIndexAttempt.java

```diff
@@ -0,0 +1,44 @@
+package io.stargate.sgv2.jsonapi.service.operation.tables;
+
+import static io.stargate.sgv2.jsonapi.util.CqlIdentifierUtil.cqlIdentifierFromUserInput;
+
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import com.datastax.oss.driver.api.core.cql.SimpleStatement;
+import com.datastax.oss.driver.api.querybuilder.SchemaBuilder;
+import com.datastax.oss.driver.api.querybuilder.schema.Drop;
+import io.stargate.sgv2.jsonapi.service.cqldriver.executor.KeyspaceSchemaObject;
+import io.stargate.sgv2.jsonapi.service.operation.SchemaAttempt;
+
+/*
+ An attempt to drop index for a column.
+*/
+public class DropIndexAttempt extends SchemaAttempt<KeyspaceSchemaObject> {
+  private final CqlIdentifier name;
+  private final boolean ifExists;
+
+  public DropIndexAttempt(
+      int position,
+      KeyspaceSchemaObject schemaObject,
+      CqlIdentifier name,
+      boolean ifExists,
```

Comment:
The pattern for options is to use the CQLOption and CQLOptions class, lets use those, have the Attempt builder create them, pass it into the attempt

Reply:
Need discussion on this. This `CQLOption` looks like we need to add a component for every command. IMO need to revisit this.